### ISSUE TITLE
Use CSS variables for customizing docs theme colors easily.

### ISSUE THREAD

**Issue by fabiocaccamo** [twisted/pydoctor#520]

**Comment by tristanlatr** [comment]
Hi @fabiocaccamo,

Could you write a couple of sentences explaining what is the issue you are facing?

The HTML include variety of css classes already, and customization is definitely possible (see #391).

If you’d like to contribute adding and documenting explicit css class for customization, that would be appreciated. But I’m unsure what are the html components that we should include in scope for such addition.

**Comment by fabiocaccamo** [comment]
Hello @tristanlatr, sorry for having not explained it.

**This is an enhancement, not a bug/problem.**

Looking how to customise the style, I expected to find some CSS variables to override defined all in the same place, instead there are many classes that define the style. 

For making a custom theme all the classes should be overridden, this is not the best/modern approach at all.

**Comment by tristanlatr** [comment]
Ok thanks I get it now. 

You're proposing to create [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) for all colors used accros the CSS, right ? Is there other stuff that would be worth adding as CSS variable ?

Do you want to work on this ?

**Comment by fabiocaccamo** [comment]
Yes, exactly. I think that starting only with colors variables would be already a good thing, other improvements may come in a second step.

If I find some time I will do it.
